aeron-driver/src/main/java/uk/co/real_logic/aeron/driver/DriverConductor.java

```diff
@@ -683,6 +684,11 @@ private void onRemoveSubscription(final long registrationId, final long correlat
         final int refCount = channelEndpoint.decRefToStream(subscription.streamId());
         if (0 == refCount)
         {
+            connections
```

Comment:
This a little bit weird. But [here](https://github.com/real-logic/Aeron/blob/master/aeron-driver/src/main/java/uk/co/real_logic/aeron/driver/NetworkConnection.java#L264) is where a little is explained. Having the conductor set the state for this creates some cache traffic on the Receiver thread.